chapters/gamestructure.adoc

```diff
@@ -97,6 +97,10 @@ The <<Referee, referee>> tosses a coin with both <<Robot Handler, robot handlers
 ==== Choosing Keeper Id
 The <<Referee, referee>> asks both <<Robot Handler, robot handlers>> which robot they will use as the keeper and forwards this information to the <<Game Controller Operator, game controller operator>>.
 
+The keeper id can also be changed during game play. This can be done automatically by using the <<Game Controller, game controller>> network interface or by asking the <<Game Controller Operator, game controller operator>> to change it in the <<Game Controller, game controller>>. The <<Game Controller Operator, game controller operator>> must not change the keeper id until the ball is at a valid position.
+
+Teams can automatically change their keeper id when the ball is on the opponent's half by using the <<Game Controller, game controller>> network interface.
```

Comment:
It is not clear, that this also refers to the manual change. What about:

> The keeper id may only be changed when ...

Reply:
Or alternatively merge with the section before:

> The keeper id can also be changed during game play, as long as the ball is in the opponent's half. This can be done automatically by using the <<Game Controller, game controller>> network interface or by asking the <<Game Controller Operator, game controller operator>> to change it in the <<Game Controller, game controller>>.

Should be clear then, that the condition applies to both automatic and manual change.

Maybe add a note, that the team should only ask the GCO when the condition is met and that the GCO is responsible for complying to the rules. I think, that's the best procedure.

Reply:
I think I got it, see if it is clearer now